eng/pipelines/runtimelab/runtimelab-post-build-steps.yml

```diff
@@ -7,53 +7,8 @@ parameters:
 
 steps:
   # Build coreclr native test output
-  - script: $(Build.SourcesDirectory)/src/coreclr/build-test$(scriptExt) skipstressdependencies skipmanaged skipgeneratelayout $(buildConfigUpper) ${{ parameters.archType }}
-    displayName: Build native test components
+  - script: $(Build.SourcesDirectory)/src/coreclr/build-test$(scriptExt) skipstressdependencies $(buildConfigUpper) ${{ parameters.archType }}
```

Comment:
This will have conflict with #86. These scripts were just moved around dotnet/runtime.

Reply:
Thanks for the heads up! Yeah, conflicts in infra/build seem inevitable - they churn _a lot_.